custom_code/hover_menu.js

```diff
@@ -0,0 +1,195 @@
+/**
+This is a script to allow cobrowsing for site navigation bar created using the "Sky Mega Menu"
+solution (http://voky.com.ua/showcase/sky-mega-menu/) in which dropdowns open and close based
+on CSS rules using the `:hover` pseudo-class.
+
+Pseudo-classes can not triggered programmatically - it has to be an actual user interaction.
+This means we cannot mirror the act of an user hovering a menu element to the other
+cobrowsing participant.
+
+This can be solved by adding an additional method (next to CSS) for opening and closing
+menu items. Namely, we can change menu items' styles based on user's mouse movements.
+This works because element style changes are caught by cobrowser and sent to the operator.
+
+Beware that this script is dependent on the structure of the menu and has been written
+solely based on the structure of a certain menu.
+**/
+
+// Helper function to query from direct children.
+var queryDirectChildren = function (parent, selector) {
```

Comment:
For visibility, there is `:scope` pseudo-selector with which you could do 
```
element.querySelectorAll(`:scope > ${selector}`)
```
, but that selector is in working draft stage and not supported everywhere.

Reply:
Yeap, also found it but didn't use it for the same reason

> but that selector is in working draft stage and not supported everywhere.